app/graphql/types/setting_type.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Types
+  class SettingType < Types::BaseObject
+    field :display_only_summary, String, null: true
```

Comment:
maybe better Boolean, because it is `true/false`?

Reply:
should work properly

<img width="320" alt="Bildschirmfoto 2019-10-18 um 13 23 30" src="https://user-images.githubusercontent.com/1942953/67090488-833c1d00-f1aa-11e9-8522-74c520a8b15c.png">